frontend/src/styles/_uswds-theme.scss

```diff
@@ -18,6 +18,10 @@ in the form $setting: value,
   // Ensure utility classes always override other styles
   $utilities-use-important: true,
 
+  // Prettier defaults for typography
+  $theme-font-type-sans: "source-sans-pro",
+  $theme-global-link-styles: true,
+
```

Comment:
Oof! I did it again, didn't mean to imply this font. We can nix this. 

> We don't have to change this; it wasn't an intentional design choice. I didn't realize that the [Truss USWDS Figma](https://www.figma.com/community/file/836611771720754351) assets I copied the components from used this font instead of what USWDS defaults to.
> – https://github.com/HHS/grants-equity/pull/371#discussion_r1294960549